mirascope/wandb.py

```diff
@@ -200,7 +261,7 @@ class Book(BaseModel):
         author: str
 
 
-    class BookRecommender(WandbOpenAIExtractor[Book]):
+    class BookRecommender(OpenAIExtractor[Book], WandbExtractorMixin[Book]):
```

Comment:
Does `WandbExtractorMixin[OpenAIExtractor[Book]]` make more sense?

Reply:
Isn't possible. The mixin is adding a method to the `OpenAIExtractor[Book]` class. The reason we have `WandbExtractorMixin[Book]` is to type the `extract_with_trace` function with the correct return type.